packages/common-scripts/src/anyone_can_pay.ts

```diff
@@ -74,23 +76,46 @@ export class CellCollector {
   }
 }
 
-export async function setupInputCell(
+export function setupInputCell(
```

Comment:
Even though we might not be using it, let's keep the API interface of setupInputCell an async function for potential needs.

Reply:
updated.